services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -0,0 +1,236 @@
+import React from "react";
+import "filepond-polyfill";
+import { FilePond, registerPlugin } from "react-filepond";
+import { notification } from "antd";
+import { invert, uniq } from "lodash";
+// import FunnelPlotOutlined from "@ant-design/icons/FunnelPlotOutlined";
```

Comment:
this is pretty much the CORE version with flush commented out because the mp3 files require a loader and it was a whole thing.